src/views/ApartmentApp/apartmentApp.scss

```diff
@@ -1,84 +1,87 @@
 @import '../../vars';
 
-strong.over-emphasized {
-  text-transform: uppercase;
-  text-decoration: underline;
-}
+.student-apartment-application,
+.apartment-application {
```

Comment:
Is there a reason that you specified both of these classes? Could it not just be `.student-apartment-application`?

Reply:
the other one occurs in a different place in the tree, and I don't remember why... I'll have to explore later the figure that out, because yes it should be fine with just `student-apartment-application`

Reply:
I'm fairly certain that `apartment-application` is applied to an otherwise empty `div` wrapping all other contents of the StudentApplication component.

Reply:
I think so too, I think it was an old leftover from many months ago

Reply:
That div also wraps up the snackbar, which is best to keep it outside the Grid otherwise it will make the Grid items sometimes align incorrectly